src/Kraken.cpp

```diff
@@ -7,14 +7,14 @@
 
 #include <czmq.h>
 #include <g2log.hpp>
-
 #include "Kraken.h"
 
 
 /// Constructing the server/Kraken that is about to be connected/impaled by the client/Harpoon
 Kraken::Kraken():
 mLocation(""), 
-mQueueLength(10),
+mQueueLength(1),
+mMaxChunkSize(10485760), //10MB
```

Comment:
OK. Just change the name of the function so that its obvious it's a setter
an d not a getter and it's fine.

**Contact Information**
*Mobile:  +1 - 617.378.1915*

*E-mail: Hedstrom@KjellKod.cc*


On Fri, Jan 16, 2015 at 1:54 PM, Ryan Kophs <notifications@github.com>
wrote:

> In src/Kraken.cpp
> <https://github.com/weberr13/QueueNado/pull/11#discussion-diff-23108676>:
>
> >  #include "Kraken.h"
> >
> >
> >  /// Constructing the server/Kraken that is about to be connected/impaled by the client/Harpoon
> >  Kraken::Kraken():
> >  mLocation(""),
> > -mQueueLength(10),
> > +mQueueLength(1),
> > +mMaxChunkSize(10485760), //10MB
>
> This just gives the user the ability to change it to something else. 10MB
> is fine, but if they want it to be 1K for example, they can do that as
> well. I used this in testing as well (set it to 1K to ensure that tons of
> packets being sent doesn't affect performance).
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/weberr13/QueueNado/pull/11/files#r23108676>.
>